controllers/front/ProductController.php

```diff
@@ -1511,6 +1517,32 @@ private function getProductPageTitle(array $meta = null)
         return $title;
     }
 
+    /**
+     * Get product quantity currently in cart.
+     *
+     * @param ProductLazyArray $product
+     *
+     * @return int
+     */
+    private function getProductQuantityInCart(ProductLazyArray $product)
```

Comment:
On my phone rn, but isnt there a function that does this?

Reply:
Yeah, `getProductQuantity` in `cart` class!
I will use that instead of my custom function ;)

Reply:
Reviewing on the phone? Sounds fun!

Reply:
What if you directly call $this->context->cart->getProductQuantity($product->getId(), $product['id_product_attribute']['quantity']?